.github/workflows/e2e.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           key: node_modules_${{ runner.os }}_${{ hashFiles('**/yarn.lock') }}
```

Comment:
In our experience it's much more efficient to cache the Yarn cache instead of the `node_modules` folder.
Example here: https://github.com/babel/babel/blob/d3a7cd5e8d5f99fb0b465a5c1d7209d5e1e6ab03/.github/workflows/ci.yml#L16-L25